src/med_bench/estimation/coefficient_product.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+from sklearn.linear_model import RidgeCV
+
+from med_bench.estimation.base import Estimator
+from med_bench.utils.constants import ALPHAS, CV_FOLDS, TINY
+from med_bench.utils.decorators import fitted
+
+
+class CoefficientProduct(Estimator):
+
+    def __init__(self, clip : float, trim : float, regularize : bool, **kwargs):
```

Comment:
Should clip and trim be provided fora ll estimators. Actually, they're not used in the CoefficientProduct